Makefile

```diff
@@ -154,6 +173,15 @@ push-manifest:
 	docker manifest push --purge $(IMAGE_TAG)
 	docker manifest inspect $(IMAGE_TAG)
 
+.PHONY: push-conformance-manifest
+push-conformance-manifest: docker-buildx-builder build-e2e-test
+	docker buildx build \
+	--no-cache \
+	--push \
+	--platform=${CONFORMANCE_ARCH} \
```

Comment:
Parallel build for multi-arch takes a long time and we have had to disable for few other projects. Could we instead use a for loop for all linux arch defined here: 

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/cdfff9350e34ce1acc2a108738d0eca2d94f9886/Makefile#L36

and build and push the manifest?

Reply:
I updated it. But out of curiosity, what's the max acceptable build time? Also with for loop approach we are pushing separate arch image to registry, however with other approach they are combine in the single blob. Wouldn't this be more appropriate?

Reply:
> what's the max acceptable build time

There is no set value for it. We tried doing this in pod identity and it took ~50m for an image build to complete with multi-arch. Switching to individual builds reduced the time to ~20m. 

> Also with for loop approach we are pushing separate arch images to registry

The individual images are then packages using `docker manifest`. I don't have strong opinion on single blob vs multiple images, just the build time from what we have observed.